TypeScript definition reference `dist` dir not bundled in the npm

Errors during compilation:

```
node_modules/svelte/store.d.ts:1:15 - error TS2307: Cannot find module './dist/store'.

1 export * from './dist/store';
                ~~~~~~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "module": "index.mjs",
   "main": "index",
   "files": [
+    "dist",
     "compiler.js",
     "register.js",
     "index.*",

